app/controllers/registration/registration.controller.js

```diff
@@ -28,6 +31,39 @@ async function showPasswordPage (req, res, next) {
   }
 }
 
+async function submitPasswordPage (req, res, next) {
+  const sessionData = req.register_invite
+  if (!registrationSessionPresent(sessionData)) {
+    return next(new RegistrationSessionMissingError())
+  }
+
+  const password = req.body['password']
```

Comment:
'password' and 'repeat-password' feel like they could be constants to 
* describe what they actually are
* make sure they're the same everywhere they're used

(these could optionally be passed in and used in the template but that might be overkill)

Something like?
```js 
const PASSWORD_INPUT_FIELD_NAME = 'password'
const REPEAT_PASSWORD_INPUT_FIELD_NAME = 'repeat-password'
```

Reply:
good point, have updated to use constants